index.html

```diff
@@ -0,0 +1,166 @@
+        </ul>
+    </header>
+    <main>
+        <section class="hero-section">
+            <div class="container-main">
+               <div class="container-start">
+                <div class="context">
+                    <img class="hamburger" src="./assets/mobile/hamburger.png" alt="menu">
+                    <p class="never">"Never forget who was there for you in difficult times"</p>
+                    <h1>Lets Learn Quran and Islamic Studies online</h1>
+                </div>
+                <div class="bg-color">
+                    <p>Al-Noor Academy is an accredited member school of the New England Association of Schools and Colleges (NEASC).</p>
+                </div>
+                <div class="date">
+                    <p class="big"><strong>2022.09.19(Monday) ~ 21(Thursday)</strong></p>
+                    <p class="small">@ AlNoor masjid Almaty, Qazaqstan<br> Akkent 56-66 </p>
+                </div>
+        </div> 
+        </section>
+        <hr>
+
+        <section>
+            <div class="container-main">
+                <div class="container-start bg-0">
+                    <div class="bg-color1">
+                    <h2 class="title">Our Online Islamic Courses</h2>
+                    <hr class="hr-org">
+                    <ul class="grey cards">
+                    <li class="pos-rel">
+                        <img class="img1" src="./assets/mobile/al-noor.png" alt="mosque" width="30px" height="30px">
+                        <h4>Foundation Course</h4>
+                        <p class="intro">Do you like learning Quran but you can not read Arabic alphabet yet? Then this is the perfect course for you. Its never too late to start learning to read the Quran!</p>  
+                    </li>
+
+                    <li class="pos-rel">
+                        <img class="img2" src="./assets/mobile/al-noor.png" alt="" width="30px" height="30px">
+                        <h4>Quran Recitation Course</h4>
+                        <p class="intro">This course is designed to improve reading & recitation of the Holy  Quran students. And to attain the Itqan level in Learn Tajweed Online with continuous Sanad connected to Prophet Muhammad (PBUH).</p>
+                    </li>
+
+                    <li class="pos-rel">
+                        <img src="./assets/mobile/al-noor.png" alt="" width="30px" height="30px">
+                        <h4>Quran Memorization Course</h4>
+                        <p class="intro">This course is designed to attain the Itqan level in Hifz Quran online through one-on-one Classes. The students (adults only) can learn advanced topics in Qiraat with objective to attain perfection of Quran memorization (Itqan).</p>
+                    </li>
+
+                    <li class="pos-rel">
+                        <img src="./assets/mobile/al-noor.png" alt="" width="30px" height="30px">
+                        <h4>Islamic Studies Course</h4>
```

Comment:
- [ ] Your design doesn't match the original design. Please make sure you follow the design. 
Cloned: 
![image](https://user-images.githubusercontent.com/58504655/192089349-f9b4d5ee-96d6-48fb-b538-619e2bea041f.png)

Original
![image](https://user-images.githubusercontent.com/58504655/192089321-c119a911-6209-4b70-a103-61aa23d0d135.png)

Reply:
Thanks again but let me mention this again **Small adjustments like text or image changes are acceptable.** And about the height if you realize text content is long that's why its looks so!

Reply:
- [ ] [optional] It would be nice if the description was shorter to make the design more applying also if you could use different images.